src/structarray.jl

```diff
@@ -425,6 +425,12 @@ function Base.copyto!(I::StructArray, doffs::Integer, J::StructArray, soffs::Int
     return I
 end
 
+function Base.fill!(I::StructArray{T}, x) where {T}
```

Comment:
Just cosmetic, but can we use `s` as a variable name for the `StructArray` in this method? I realize that the source code is not fully consistent, but it'd be good to standardize this.

Reply:
Updated